docs/csharp/async.md

```diff
@@ -240,7 +240,7 @@ With async programming, there are some details to keep in mind that can prevent
 
 * **Consider using** `ValueTask` **where possible**
 
-  Returning a `Task` object from async methods can introduce performance bottlenecks in certain paths. `Task` is a reference type, so using it means allocating an object. In cases where a method declared with the `async` modifier returns a cached result or completes synchronously, the extra allocations can become a significant time cost in performance critical sections of code. It can become costly if those allocations occur in tight loops. For more information, see [generalized async return types](whats-new/csharp-7.md#generalized-async-return-types).
+  Returning a `Task` object from async methods can introduce performance bottlenecks in certain paths. `Task` is a reference type, so using it means allocating an object. In cases where a method declared with the `async` modifier returns a cached result or completes synchronously, the extra allocations can become a significant time cost in performance critical sections of code. It can become costly if those allocations occur in tight loops. For more information, see [generalized async return types](whats-new/csharp-version-history.md#c-version-70).
```

Comment:
The "for more information" part doesn't seem accurate - because that section https://review.docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-version-history?branch=pr-en-us-27112#c-version-70 doesn't contain anything about async return types.

Reply:
Ah ha!  There's a reason I've started documenting the new features in the reference and conceptual docs and pointing to them from the "what's new" article. It makes these edits easier.  

For the "generalized async returns", the link should be `csharp/language-reference/keywords/async#return-types`
For the "async Main", it should be fundamentals/program-structure/main-command-line.md#async-main-return-values, as you note below.